### DISCUSSION TITLE
3.4.2 "Copy image" in context menu

### DISCUSSION THREAD

**Discussion by nicamoq** [sumatrapdfreader/sumatrapdf#2615] [Q&A]
Is it possible to bring back the "Copy image" option below "Copy selection" in the context menu? There doesn't seem to be a way to copy a whole page anymore.

**Comment by kjk** [comment]
Good idea, I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/2616 to track this

**Comment by kjk** [comment]
Actually, I can't reproduce. Can you provide a file that works differently in 3.4 vs. 3.3 and exact reproduction steps in https://github.com/sumatrapdfreader/sumatrapdf/issues/2616
